lib/ui/menu_bottom_sheet.dart

```diff
@@ -37,7 +38,7 @@ class MenuBottomSheet extends StatelessWidget {
               padding: const EdgeInsets.symmetric(horizontal: 32.0),
               child: Container(
                 height: 1,
-                color: const Color(0xFFF0F0F0),
+                color:  AppColors.dividerColor,
```

Comment:
A ta podwójna spacja to po co nam?
```suggestion
                color: AppColors.dividerColor,
```